examples/online_serving/prometheus_grafana/grafana.json

```diff
@@ -593,6 +593,22 @@
           "range": true,
           "refId": "C",
           "useBackend": false
+        },
+        {
+          "datasource": {
+            "type": "prometheus",
+            "uid": "${DS_PROMETHEUS}"
+          },
+          "disableTextWrap": false,
+          "editorMode": "builder",
+          "expr": "vllm:num_requests_corrupted{model_name=\"$model_name\"}",
```

Comment:
I don't think we should include such a niche, off-by-default, metric in the default dashboard